CLONING.md

```diff
@@ -0,0 +1,217 @@
+- `REACT_APP_FIREBASE_DATABASE_URL` - same value as the value in `.env`
+- `REACT_APP_FIREBASE_PROJECT_ID` - same value as the value in `.env`
+- `REACT_APP_FIREBASE_STORAGE_BUCKET` - same value as the value in `.env`
+- `REACT_APP_FIREBASE_MESSAGING_SENDER_ID` - same value as the value in `.env`
+- `REACT_APP_FIREBASE_APP_ID` - same value as the value in `.env`
+- `REACT_APP_FIREBASE_MEASUREMENT_ID` - same value as the value in `.env`
+- `REACT_APP_HERE_API_KEY` - same value as the value in `.env`
+- `REACT_APP_PURPLEAIR_READ_API_KEY` - same value as the value in `.env`
+- `REACT_APP_PURPLEAIR_WRITE_API_KEY` - same value as the value in `.env`
+
+### Continuous Deployment
+
+We use [GitHub Actions for Firebase](https://github.com/w9jds/firebase-action) to automatically re-deploy our website whenever code is pushed to the `main` branch. This re-deploys Firebase Hosting and our Firebase Cloud Functions. To use this action, you need to add an authentication token for Firebase to GitHub Secrets with the secret name `FIREBASE_TOKEN`. As the documentation describes, this token can be aquired through the `firebase login:ci` command run locally on your computer after setting up the Firebase CLI.
+
+In GitHub, you will also need to make sure that Actions are enabled under the repository's Settings.
+
+### Dependabot
+
+If you are actively developing the app, you will want to re-enable Dependabot, which automatically creates pull requests to update the libraries that the app uses. To re-enable Dependabot, remove each of the `open-pull-requests-limit: 0` lines in `.github/dependabot.yml` file.
+
+## Local content
+
+This app was originally developed for the community of South Gate, California. Much of the South Gate specific content was externalized to a few files so that modifying the website for use with another community. However, further externalization of content specific to South Gate would be a good extension of this project.
+
+### Static Content
+
+All static content is in the `public/locales/` directory.
+
+- Most of the content in `health.json` files used by the `Health Information` page is generalizable to other communities, except for the beginning that descibes health concerns in South Gate.
+- Much of the content in the `about.json` files used by the `About` page will need to be updated for other communities.
+- In the `administration.json` files, the `accessDenied.shouldHaveAccess` key should be updated to say who should be contacted in case someone wants to be an administrator of the website.
+
+### Website Metadata
+
+- In `src/components/NavBar/Logo.tsx`, the website name needs to be updated.
+- The logo file for the South Gate CEHAT `src/media/CEHATLogo.png`, which is used in the `src/pages/About.tsx` file, should be updated to the appropriate organization logo.
+- `public/index.html` needs the website title and the description updated. The necessary lines to change are:
+
+  ```html
+  <meta
+    name="description"
```

Comment:
Put the `<title>` section before the `<meta>` section to match the order that you introduce them in the previous sentence. For the `<meta>` section, is `name` a description? How does that differ from `content`? Don't bother adding long explanations, but perhaps replace the "Website for air quality....." with a string representing what kind of info is supposed to go there:

ex:
name = "website name"
content = "brief description"

^ if that's what it's supposed to be

Reply:
Good call, I updated this to be more clear